module.json

```diff
@@ -3,7 +3,7 @@
     "name": "lmrtfy",
     "title": "Let Me Roll That For You!",
     "description": "Ask your players to make rolls for you.",
-    "version": "1.9",
+    "version": "1.9.1",
```

Comment:
This change isn't necessary, our release automation handles this. We're actually at 1.11 right now and will hit 1.12 with this release (versioning scheme is just adopted from the original author's, ultimately the incrementing number is what matters)

Reply:
Oh gotcha. I should have noticed that 1.11 was the version I had myself lol.